libs/dh/globalization/assets-localization/src/assets/i18n/en.json

```diff
@@ -1,3 +1,10 @@
 {
```

Comment:
question: how about suggesting English translation texts in this PR?

Reply:
It would be preferable, however we don't have these and the English version is not prioritized atm.

Reply:
Are we not allowed to make suggestions ourselves?

Reply:
We have done it for now, and have asked Mogens how we should handle future translations